eslint.style.config.mjs

```diff
@@ -0,0 +1,19 @@
+import typescriptEslint from "typescript-eslint";
+import stylisticJsx from "@stylistic/eslint-plugin-jsx";
+import baseConfig from "./eslint.config.mjs";
+
+// style configuration extends default/development configuration
+// TODO: document why this is separate from the base config
```

Comment:
The reason is that at the time it was felt that enforcing style rules might be too opinionated/heavy-handed for a starter project where stylistic opinions might vary among projects/developers.